packages/desktopjs-openfin/tests/openfin.spec.ts

```diff
@@ -647,72 +649,80 @@ describe("OpenFinContainer", () => {
             container.addListener("window-created", () => done());
             MockDesktop.application.emit('window-created', { name: "name" });
         });
+    });
 
-        describe("window management", () => {
-            it("getAllWindows returns wrapped native windows", (done) => {
-                container.getAllWindows().then(windows => {
-                    expect(windows).not.toBeNull();
-                    expect(windows.length).toEqual(2);
-                    expect(windows[0].innerWindow).toEqual(MockWindow.singleton);
-                    done();
-                });
+    describe("window management", () => {
```

Comment:
This entire describe block was incorrect under another describe so moved out to higher level